cmd/keymasterd/app.go

```diff
@@ -61,6 +61,7 @@ const (
 	AuthTypeIPCertificate
 	AuthTypeTOTP
 	AuthTypeOkta2FA
+	AuthTypeBootstrapOTP
```

Comment:
Since there isn't any code dealing with this bootstrap data, why include it in this PR?